sceptre/environment.py

```diff
@@ -282,12 +282,16 @@ def _check_for_circular_dependencies(self):
         self.logger.debug("Checking for circular dependencies...")
 
         if self.stacks:
+            top_level_path = self.path + '/'  \
```

Comment:
Hi @ankitanallana could I ask that you use the default python path library when working with paths - it helps ensure some level of Windows support. 

In this case something like the following might work (assuming `top_level_path` is correct):

```py
if self.path and self.path != '.':
    top_level_path = os.path.join(self.path, '')
```

Reply:
sure! this is a good point! apologize for the delay, i shall incorporate this too!